e2etest/blobtx_test.go

```diff
@@ -109,8 +109,7 @@ func TestBlobTx(t *testing.T) {
 				time.Now(),
 			},
 			expect: []actionExpect{&functionExpect{func(test *e2etest, act *action.SealedEnvelope, receipt *action.Receipt, err error) {
-				r.ErrorIs(err, action.ErrUnderpriced)
-				r.ErrorContains(err, "blob fee cap is too low")
+				r.ErrorIs(err, errReceiptNotFound)
```

Comment:
now blob fee compare is moved to `ValidateWithState()`, actpool won't return error, the tx is rejected in workingset and not included into the block